### DISCUSSION TITLE
Do GitHub Actions contribute to a repo's Traffic stats under Insights?

### DISCUSSION THREAD

**Discussion by JeffreyCA** [community/community#23806] [New to GitHub]
<p>For example, when a GitHub action runs, does it count towards a repo’s unique clone count?</p>

**Comment by yamiacat** [comment, marked as answer]
<p>It depends how the Action was authenticating! If the Action was using a personal access token, it will appear in the Insights panel as that person.</p>
<p>Otherwise it will have its own entry, and will count as one unique cloner, no matter how many times it clones the repository.</p>

**Reply by kisvegabor** [reply]
> Otherwise it will have its own entry, and will count as one unique cloner, no matter how many times it clones the repository.


Does it mean one unique cloner in the whole 14 days or on a given day?
